### ISSUE TITLE
Are `input`s of type `number` supported?

### ISSUE THREAD

**Issue by mtseD2L** [Brightspace/valence-ui-input#89]
I'm using it in https://github.com/Brightspace/manage-extensibility-ui/blob/fc255a70ed891142f2c760c097ff58c6c93ceffb/src/app.scss#L75, and want to understand if it's supported or not.

**Comment by dlockhart** [comment]
@mtseD2L: no, number inputs aren't currently supported... which is probably why they look so ugly. :) 

The challenge isn't really even making them look nice, but more around tying in our own number parsing/formatting rules since we let our users customize decimals, separators, group sizes, negative symbols, etc. We'd also need to tie in validation as well.

Certainly a valuable feature request, but non-trivial!
